bylaws/009-funding.md

```diff
@@ -0,0 +1,55 @@
+# Funding
+
+## Scope and objectives
+
+PHP-FIG is, as it often happens in Open Source communities, an organization composed by volunteers only; this doesn't exclude the fact that the organization itself may have some costs to sustain itself, for example maintaining tools such as a site and an email account. For such and similar reasons, we need a way to raise and govern money inside our organization.
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes requires a bylaw change vote. 
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][].
+
+## Fiscal hosting
+
+For handling money and fiscal matters, the PHP-FIG MUST use [Open Collective](https://opencollective.com/) as a platform and [Open Source Collective](https://opencollective.com/opensource) as a fiscal host.
+All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's OpenCollective account. They SHALL NOT use those permissions in a discretionary manner, except as stated explicitly in this bylaw.
+
+All monetary contributions MUST flow through Open Collective, so that any tax considerations will be handled automatically, and every financial decision MUST be published and publicly available here and through Open Collective's budgetary tools.
+
+# Raising money
+
+PHP-FIG SHALL NOT solicit donations from any party, especially from private individuals. The only allowed funding sources MUST be listed below.
+
+### Through OpenCollective
+
+Direct contributions MUST NOT be accepted through OpenCollective; the collective home page MUST be set up in a way that:
+ * all contribution sections and features are hidden and/or disabled, if possible;
+ * explicit wording will be visible reminding visitors of the PHP-FIG stance in the matter.
+
+### Through TideLift
+
+Contributions will be accepted through [TideLift](https://tidelift.com/). All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's TideLift account, which MUST be configured to pay out to the Open Collective account. 
+
+## Spending money
+
+Funds donated to the PHP-FIG MUST NOT be used for non-personnel operating expenses.  PHP-FIG SHALL NOT pay individual contributors to PHP-FIG standards or working groups.
```

Comment:
This sounds like an improper double-negative? _MUST NOT .. non-personnel_

Reply:
Should `non-personnel` here be `non-organizational`?

Reply:
I'm not a native speaker, but it sounded weird to me too. @Crell, WDYT? The wording was originally yours.

Reply:
Solved in d146b7b